README.md

```diff
@@ -166,10 +166,16 @@ return static function (ContainerConfigurator $containerConfigurator): void {
     // is your PHP version different from the one your refactor to? [default: your PHP version], uses PHP_VERSION_ID format
     $parameters->set(Option::PHP_VERSION_FEATURES, PhpVersion::PHP_72);
 
-    // auto import fully qualified class names? [default: false]
+    // whether Rector can leave FQCN or automatically add them to use statements and remove from code occurences [default: false]
```

Comment:
A or B option with true or false,  is like "do you have pizza or cake", yes or no.
Better leave one option.

Reply:
I'm affraid I don't understand :)

Reply:
There is a binary option with binary answer: `// whether Rector can A or B`

Reply:
I think I know what's your point. I used `or` an then `and`. i think a comma would be helpful or as we are developers some parentheses what to read first :)